manifest/manifest.go

```diff
@@ -134,8 +134,9 @@ func (m Manifest) Validate() []error {
 		mem := entry.Memory
 
 		if mem < mem_min && mem != 0 { //Memory(0) {
-			e := fmt.Errorf("%s has invalid mem_limit %#v: should be either 0, or at least %#vMB",
+			e := fmt.Errorf("%s has invalid mem_limit %#v bytes / %#v MB: should be either 0, or at least %#vMB",
```

Comment:
So the input is always in bytes? In that case I'd keep the validation message in bytes too. How about something like:

> invalid mem_limit 2048 bytes (0 MB). Should be either 0 bytes or at least 4000000 bytes (4 MB)"